tensorflow/lite/tools/make/build_riscv_lib.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash 
```

Comment:
there is an unnecessary character after 'bash'. plz remove it.

Reply:
Yes, Sure.